styles/common_styles.css

```
@@ -32,9 +32,10 @@ footer.sticky {
 }
 .meta-icon-container {
     display:grid;
-    grid-template-columns: 25% 25% 25% 25%;
+    grid-template-columns: 40% 20% 40%;
```

```suggestion
    grid-template-columns: 4fr 2fr 4fr;
```
Again, not sure if this would break the layout somehow, have not tested proposed change.